docs/development.md

```diff
@@ -71,28 +69,35 @@ Everytime you want to run bors:
 - Try `@bors ping` on some PR on the test repository :)
 
 ## Database
-You must have `sea-orm-cli` installed for the following commands to work.
+You must have `sqlx-cli` installed for the following commands to work.
 ```console
-$ cargo install sea-orm-cli
+$ cargo install sqlx-cli --no-default-features --features native-tls,postgres
```

Comment:
Please pin the `sqlx-cli` version in this command (`cargo install --version X.Y.Z`).